fluent-plugin-logdna.gemspec

```diff
@@ -17,7 +17,8 @@ Gem::Specification.new do |s|
   s.files         = `git ls-files -z`.split("\x0")
   s.executables   = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
   s.required_ruby_version = Gem::Requirement.new('>= 2.0.0')
-  s.add_development_dependency "bundler", "~> 1.16"
+  s.add_development_dependency 'bundler', '~> 1.16'
   s.add_runtime_dependency 'fluentd', '>= 0.12.0', '< 2'
   s.add_runtime_dependency 'http', '~> 2.0', '>= 2.0.3'
+  s.add_development_dependency 'rubocop', '~> 0.78'
```

Comment:
nit: can we reorder these so that runtime deps are first, followed by dev deps

Reply:
Done!